Makefile

```diff
@@ -490,6 +490,10 @@ pilot-test: pilot-agent
 istioctl-test: istioctl
 	go test -p 1 ${T} ./istioctl/...
 
+.PHONY: iptables-test
+iptables-test:
```

Comment:
nit: we already have some tests for the iptables in common-test target (added a few weeks ago I think). maybe they can be consolidated into one target, either moving those tests here or these tests to the common-test

Reply:
Fixed.